libs/errorHandler.js

```diff
@@ -3,18 +3,21 @@
 /*  Load gulp + plugins
 /* ------------------------------------------------------------------------- */
 
-var notify  = require("gulp-notify");
+var gutil = require('gulp-util');
```

Comment:
How do you mean, handling other functions? Which other functions would you like it to handle?
And yes, I prefer using gulp-util in combination with gulp-plumber..
Just for the simple reason that output is much nicer formatted, in my opinion.


> On 18 May 2015, at 19:38, Anshul Goel <notifications@github.com> wrote:
> 
> In libs/errorHandler.js <https://github.com/goel4ever/usage-gulp/pull/3#discussion_r30529631>:
> 
> > @@ -3,18 +3,21 @@
> >  /*  Load gulp + plugins
> >  /* ------------------------------------------------------------------------- */
> >  
> > -var notify  = require("gulp-notify");
> > +var gutil = require('gulp-util');
> We were using gulp-util already. It seems this error handler is formatting the output only and not handling other functions yet. Did you happen to find gulp-util more useful than gulp-notify?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/goel4ever/usage-gulp/pull/3/files#r30529631>.
>